Dev/dylan/week21

Dylan773 opened the pull request:
Changed 'DOB' property to 'RecordCreated'
* This property represents the date value the staff record was created on.
* Renamed DOB fields and methods to RecordCreated in clsStaff.
* Renamed existing test methods for DOB to RecordCreated in tstStaff.
* Changed webpage code from DOB to RecordCreated, e.g. StaffMember.DOB -> StaffMemeber.RecordCreated.

FYI: Followed the lab sheet, if there are any bad practices in the code then they cant moan as it's their way of doing things although its probably ass.

Implemented validation for the staff system.
* Checks if the user input is of the correct type and format.
* Created new test methods for checking if user input is in the acceptable range for that specific field (boundaries).

dylanbrand7 committed:
Added validation

dylanbrand7 committed:
Validation (Presentation Layer)

Ashley-exe commented on TestingStaff/tstStaff.cs:
```diff
@@ -7,6 +7,13 @@ namespace TestingStaff
     {
+
+        // TEST DATA
+        string StaffPassword = "password";
```
These are static constants, and have no accessability modifier.

* Public / private / protected
* Static
* readonly
* SNAKE_CASE_NAMES (ctrl + r twice on a symbol to refactor it)

Ashley-exe commented on TestingStaff/tstStaff.cs:
```diff
@@ -133,7 +142,7 @@ public void TestStaffIDFound()
 
             // Creating some test data to use with the method.
-            Int32 StaffID = 12;
+            int StaffID = 12;
```
Good simplification to raw

Ashley-exe reviewed (changes requested):
Looks good, just requesting a quick change of some modifiers. 5 minute job.

Dylan773 committed:
Updated Accessability

Have decided to put them as public for the time being, will change to private once I can run the tests on the vm.

Ashley-exe reviewed (approved):
Whoops, completely missed the e-mail for this. Sorry it took so long!

Thanks for making those changes 😄
